Mathlib/Analysis/Analytic/Linear.lean

```diff
@@ -27,30 +27,6 @@ noncomputable section
 
 namespace ContinuousLinearMap
 
-/-- Formal power series of a continuous linear map `f : E →L[𝕜] F` at `x : E`:
-`f y = f x + f (y - x)`. -/
-def fpowerSeries (f : E →L[𝕜] F) (x : E) : FormalMultilinearSeries 𝕜 E F
-  | 0 => ContinuousMultilinearMap.curry0 𝕜 _ (f x)
-  | 1 => (continuousMultilinearCurryFin1 𝕜 E F).symm f
-  | _ => 0
-#align continuous_linear_map.fpower_series ContinuousLinearMap.fpowerSeries
-
-theorem fpower_series_apply_zero (f : E →L[𝕜] F) (x : E) :
-    f.fpowerSeries x 0 = ContinuousMultilinearMap.curry0 𝕜 _ (f x) :=
-  rfl
-
-theorem fpower_series_apply_one (f : E →L[𝕜] F) (x : E) :
-    f.fpowerSeries x 1 = (continuousMultilinearCurryFin1 𝕜 E F).symm f :=
-  rfl
-
-theorem fpowerSeries_apply_add_two (f : E →L[𝕜] F) (x : E) (n : ℕ) : f.fpowerSeries x (n + 2) = 0 :=
-  rfl
-#align continuous_linear_map.fpower_series_apply_add_two ContinuousLinearMap.fpowerSeries_apply_add_two
-
-attribute
-  [eqns fpower_series_apply_zero fpower_series_apply_one fpowerSeries_apply_add_two] fpowerSeries
-attribute [simp] fpowerSeries
-
```

Comment:
I moved this to `FormalMultlinearSeries.lean` (it is purely algebraic so it seems to belong better there)